VS2017/Source/tests/TestLight.cpp

```diff
@@ -29,6 +30,9 @@ namespace test {
 
 		GLCall(glEnable(GL_CULL_FACE));
 		glEnable(GL_DEPTH_TEST);
+
+		unsigned int uint = 5;
+		std::cout << uint / 2 << std::endl;
```

Comment:
is there a purpose for this or was it just for testing?

Reply:
Just testing, nice catch, removed